process/wsindexer/indexer.go

```diff
@@ -49,6 +53,16 @@ func (i *indexer) GetOperationsMap() map[data.OperationType]func(d []byte) error
 	}
 }
 
+func (i *indexer) ProcessPayload(payload *data.PayloadData) error {
+	function, ok := i.actions[payload.OperationType]
```

Comment:
might rename `function` to `operationAction`

Reply:
changed